bigbluebutton-web/src/java/org/bigbluebutton/api/messaging/Constants.java

```diff
@@ -31,6 +31,7 @@
   public static final String RECORDING                       = "recording";
   public static final String AUTO_START_RECORDING            = "auto_start_recording";
   public static final String ALLOW_START_STOP_RECORDING      = "allow_start_stop_recording";
+  public static final String WEBCAMS_ONLY_FOR_MODERTORS      = "webcams_only_for_moderators";

```

Comment:
All of the other instances leave off the "s" at the end. Should probably remove to match.